python/example_code/bedrock-runtime/models/ai21_labs_jurassic2/invoke_model.py

```diff
@@ -26,8 +26,13 @@
 # Convert the native request to JSON.
 request = json.dumps(native_request)
 
-# Invoke the model with the request.
-response = client.invoke_model(modelId=model_id, body=request)
+try:
+    # Invoke the model with the request.
+    response = client.invoke_model(modelId=model_id, body=request)
+
+except Exception as e:
```

Comment:
```suggestion
except (ClientError, Exception) as e:
```

Reply:
Added explicit handling of ClientError to all examples.